src/test/scala/mesosphere/mesos/TaskGroupBuilderTest.scala

```diff
@@ -22,6 +25,10 @@ import scala.collection.JavaConverters._
 import scala.collection.breakOut
 
 class TaskGroupBuilderTest extends UnitTest with Inside {
+
+  implicit val clock = Clock.systemUTC()
+  val config = AllConf.withTestConfig("--draining_seconds", "300")
```

Comment:
why this? isn't the feature disabled by default?

Reply:
done